backend/src/apiserver/main.go

```diff
@@ -106,12 +106,23 @@ func main() {
 	}
 	log.SetLevel(level)
 
+	go reconcileSwfCrs(resourceManager)
+
 	go startRpcServer(resourceManager)
 	startHttpProxy(resourceManager)
 
 	clientManager.Close()
 }
 
+func reconcileSwfCrs(resourceManager *resource.ResourceManager) {
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
+	defer cancel()
+	err := resourceManager.ReconcileSwfCrs(ctx)
+	if err != nil {
+		log.Errorf("Could not reconcile the ScheduledWorkflow Kubernetes resources: %v", err)
```

Comment:
I agree with your approach of not exiting with an error code in this case, but I'm curious what others think. My concern is that it'd be hard to know that this failed as a KFP admin but it also doesn't seem warranted to keep the API server from running if this can't succeed.